app/models/katello/pool.rb

```diff
@@ -2,14 +2,22 @@ module Katello
   class Pool < Katello::Model
     include Katello::Authorization::Pool
 
-    attr_accessor :quantity_attached
+    def quantity_attached
+      if activation_keys.present?
+        activation_keys.first.get_key_pools
```

Comment:
I'm don't think we can use .first here as Pools to ActivationKeys is a many to many relationship. A pool can have many activation_keys. Getting the 'quantity_attached' field was a somewhat complicated issue that required us to create [ActivationKeySubscriptonPresenter](https://github.com/Katello/katello/blob/master/app/presenters/katello/activation_key_subscription_presenter.rb)